src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -810,8 +814,16 @@ void GcsPlacementGroupManager::CleanPlacementGroupIfNeededWhenJobDead(
     }
   }
 
-  for (const auto &group : groups_to_remove) {
-    RemovePlacementGroup(group, [](Status status) {});
+  for (const auto &placement_group_id : groups_to_remove) {
+    RemovePlacementGroup(placement_group_id, [placement_group_id](Status status) {
+      if (status.ok()) {
+        RAY_LOG(INFO) << "Placement group of an id, " << placement_group_id
+                      << " is removed successfully for job died.";
+      } else {
+        RAY_LOG(ERROR) << "Failed to remove the placement group " << placement_group_id
```

Comment:
```suggestion
        RAY_LOG(WARNING) << "Failed to remove the placement group " << placement_group_id
```

ERROR will be printed to the user driver, so we should avoid using it unless this is something user-facing.

Reply:
I fixed all comments below.